csharp/SparkCLR.linux.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.SparkCLR</id>
+    <version>1.0.0</version>
+    <authors>Microsoft Corporation</authors>
+    <owners>Microsoft Corporation</owners>
+    <licenseUrl>https://github.com/Microsoft/SparkCLR/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/SparkCLR</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>C# language binding and extensions to Apache Spark</description>
+    <releaseNotes>See release notes on GitHub https://github.com/Microsoft/SparkCLR/releases</releaseNotes>
+    <copyright>Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.</copyright>
+    <tags>C# .NET Apache Spark RDD DataFrame</tags>
+    <dependencies>
+      <dependency id="Razorvine.Pyrolite" version="4.10.0.0" />
+      <dependency id="Razorvine.Serpent" version="1.12.0.0" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="Adapter/Microsoft.Spark.CSharp/bin/Release/Microsoft.Spark.CSharp.Adapter.dll" target="lib/net45" />
+    <file src="Worker/Microsoft.Spark.CSharp/bin/Release/CSharpWorker.exe" target="lib\net45" />
```

Comment:
Nit: lib/net45 
[ie. forward slash]
But I guess nuget seem to work with both / either these days. :)